iron-a11y-keys-behavior.html

```diff
@@ -130,14 +84,12 @@
       var validKey = '';
       if (key) {
         var lKey = key.toLowerCase();
-        if (lKey.length == 1) {
-          if (KEY_CHAR.test(lKey)) {
```

Comment:
it was limited to alphanumeric chars, but the requirement is to support also special chars like ! @# etc..